Flow.Launcher.Core/ExternalPlugins/Environments/JavaScriptEnvironment.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using Flow.Launcher.Infrastructure.UserSettings;
+using Flow.Launcher.Plugin;
+
+namespace Flow.Launcher.Core.ExternalPlugins.Environments
+{
+
+    internal class JavaScriptEnvironment : TypeScriptEnvironment
```

Comment:
Mainly so we don't need to reimplement anything since they both are using same everything. So plugin dev can set their plugin type as JavaScript or Typescript.